app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -281,13 +295,63 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
             actions: ["transcribe:StartStreamTranscriptionWebSocket","transcribe:StartMedicalStreamTranscription","comprehendmedical:InferICD10CM","comprehendmedical:InferRxNorm","comprehendmedical:DetectEntitiesV2"]
           })
         );
+        
+        // Dynamodb
+        const TableSessions = new ddb.Table(this, 'TableSessions', {
+          tableName: 'Sessions',
+          partitionKey: { name: 'PatientId', type: ddb.AttributeType.STRING },
+          sortKey: {name: 'SessionId', type: ddb.AttributeType.STRING },
+          serverSideEncryption: true
+        });
+
+        TableSessions.addGlobalSecondaryIndex({
+          indexName: "hcpIndex",
+          partitionKey: { name: 'HealthCareProfessionalId', type: ddb.AttributeType.STRING },
+          sortKey: {name: 'SessionId', type: ddb.AttributeType.STRING }
+        });
+        
+        const TablePatients = new ddb.Table(this, 'TablePatients', {
+          tableName: 'Patients',
+          partitionKey: { name: 'PatientId', type: ddb.AttributeType.STRING },
+          serverSideEncryption: true
+        });
+        
+        const TableHealthCareProfessionals = new ddb.Table(this, 'TableHealthCareProfessionals', {
+          tableName: 'HealthCareProfessionals',
+          partitionKey: { name: 'HealthCareProfessionalId', type: ddb.AttributeType.STRING },
+          serverSideEncryption: true
+        });
 
         // Lambda
+        /* CreatHealthCarePro API Processor */
```

Comment:
Please remove out commented code

Reply:
It is Done. https://github.com/aws-samples/medical-transcription-analysis/blob/cb201f940cf14ebf98d9d637d7a2c2e96cc6dba3/app/lib/medical-transcription-analysis-stack.ts#L325-L327